### DISCUSSION TITLE
I’d set up ELAN_MAC_ADDR without setting up MAC_KEY

### DISCUSSION THREAD

**Discussion by zentavr** [Anime4000/RTL960x#139] [Q&A]
I have ODI DFP-34X-2C2 device and tried to clone my huaweii 1GE-XPON-ONU.

I put:
```

flash set GPON_PLOAM_FORMAT 0
flash set GPON_PLOAM_PASSWD 61343330393135377878
flash set ELAN_MAC_ADDR 249EBAC7B7CA
flash set OMCI_SW_VER1 V3R017C00S100
flash set OMCI_SW_VER2 V3R017C00S100
flash set GPON_ONU_MODEL 1GE-XPON-ONU
flash set GPON_SN HWTCBAC7B7CA
flash set PON_VENDOR_ID HWTC
flash set HW_HWVER 6A5.A
flash set HW_CWMP_MANUFACTURER 'Huawei Technologies Co., Ltd'
flash set HW_CWMP_PRODUCTCLASS HGU

```

and did not set up MAC_KEY here. How can I restore the device?

**Comment by zentavr** [comment]
What I can see from my router is that the sticks IP appears for 1-2 pings and disappears. Seems like stick is in the endless reboot loop.

**Reply by Anime4000** [reply]
only way to restore device is via UART

**Reply by zentavr** [reply]
As I understood - I need to connect two cables to these 2 "dots" at the right bottom of the board and the third one should be connected to the case in order to make UART work, correct?

![image](https://user-images.githubusercontent.com/595334/228169394-29d2cc0d-b73a-4896-b273-23b6955d41fa.jpeg)

**Reply by zentavr** [reply]
By the way, I had the second device where I had put the MAC_KEY, but it's a brick as well. Seems like the only way to check what's going on is to connect it to UART as well.

**Reply by Anime4000** [reply]
UART pin is here:
https://github.com/Anime4000/RTL960x/blob/main/Docs/UART.md

the 3rd wire is ground, need to attach at SFP host body

**Reply by zentavr** [reply]
@Anime4000 yeah, I saw exactly that manual. My board is very similar to what **RTL9601D**, but I don't have that holes at the contact fields.

**Reply by Anime4000** [reply]
just solder wire with the pad

**Comment by dylangerdaly** [comment]
Pretty sure this happened to me, I have the exact same device.

@zentavr how'd you restore once you hooked up UART?

**Reply by zentavr** [reply]
@dylangerdaly my device provided an IP for the very short period of time. As I recall, I’d sent my steps to the document with the scripts I’d used.

**Reply by dylangerdaly** [reply]
Oh so you didn't need to solder UART in the end?

**Reply by zentavr** [reply]
@dylangerdaly #143 - check this one

**Reply by zentavr** [reply]
@dylangerdaly the discussion was in #84
